src/containers/guest-home-page/WhatCanYouDo.tsx

```diff
@@ -3,62 +3,55 @@ import { useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import InfoCard from '~/components/info-card/InfoCard'
-
 import SignupDialog from '~/containers/guest-home-page/signup-dialog/SignupDialog'
+
 import { useModalContext } from '~/context/modal-context'
-import { student, tutor } from '~/constants'
 import { guestRoutes } from '~/router/constants/guestRoutes'
-
 import learnImg from '~/assets/img/guest-home-page/learnImg.png'
 import teachImg from '~/assets/img/guest-home-page/teachImg.png'
 
+import { UserRoleEnum } from '~/types'
 import { styles } from '~/containers/guest-home-page/styles/WhatCanYouDo.styles'
 
-const sectionId = guestRoutes.navBar.whatCanYouDo.route
-
 const cardData = [
   {
-    id: 1,
     img: learnImg,
     title: 'guestHomePage.whatCanYouDo.learn.title',
     description: 'guestHomePage.whatCanYouDo.learn.description',
     actionLabel: 'guestHomePage.whatCanYouDo.learn.actionLabel',
-    actionType: student
+    actionType: UserRoleEnum.Student
   },
   {
-    id: 2,
     img: teachImg,
     title: 'guestHomePage.whatCanYouDo.teach.title',
     description: 'guestHomePage.whatCanYouDo.teach.description',
     actionLabel: 'guestHomePage.whatCanYouDo.teach.actionLabel',
-    actionType: tutor
+    actionType: UserRoleEnum.Tutor
   }
 ]
 
 const WhatCanYouDo = () => {
   const { t } = useTranslation()
   const { openModal } = useModalContext()
 
-  const openDialog = (type) => {
+  const openDialog = (type: string) => {
```

Comment:
as i have seen type value can be only one of user roles so maybe type it with role enum or united type of roles, how do you think?

Reply:
indeed. fixed